user/urls.py

```diff
@@ -7,4 +13,7 @@
     path("register/", CreateUserView.as_view(), name="create"),
     path("login/", CreateTokenView.as_view(), name="login"),
```

Comment:
We don't need this endpoint cause we use path("token/", TokenObtainPairView.as_view(), name="token_create").
Also, you can delete AuthTokenSerializer and CreateTokenView.